EnRonSwanson/BangazonCLI
# can add customer

## Fixes #1.

## Changes proposed in this pull request: 
- Can now add a customer to the database
- modified the customer table to better fit the schema


## How to test: 
- comment out any code that has squiggles 
- delete your local db file and close out of db browser
- enter dotnet run and follow the instructions
- check out the database in db browser


@EnRonSwanson/
